### DISCUSSION TITLE
Script in infinite loop

### DISCUSSION THREAD

**Discussion by WoojinJung-04** [mylinuxforwork/dotfiles#468] [Q&A]
![image](https://github.com/user-attachments/assets/37c61e61-0dc2-467b-ab75-d606ca148c2f)
The script fell in a seemingly infinite loop. I want to know if this is solvable from my side.

I would appreciate any effort anyone can give me. Thanks in advance.

Device : arch linux /w Thinkpad X230

**Comment by WoojinJung-04** [comment]
Rebooting the computer solved the issue!! No attention needed!

**Comment by mylinuxforwork** [comment]
Great
